src/Miunie.Discord/Embeds/EmbedConstructor.cs

```diff
@@ -75,6 +75,11 @@ public static Embed ToEmbed(this MiunieGuild mGuild, ILanguageProvider lang)
                 .AddField(lang.GetPhrase(PhraseKey.GUILD_EMBED_CREATED_AT_TITLE.ToString()), $"{mGuild.CreationDate:d} at {mGuild.CreationDate:t} UTC")
                 .Build();
 
+        private static string ToPossessiveForm(this string s)
+            => s.Last() == 'S'
+                ? s + "'"
+                : s + "'S";
```

Comment:
Do we really need this? Where is the value-added?

Reply:
After reading some previously overlooked reviews/comments this is what Peter is looking for.